### ISSUE TITLE
Using HTTPS options does not work, causes crash

### ISSUE THREAD

**Issue by chase-smith** [alm-tools/alm#339]
The options --httpskey and --httpscert do not work, and crash alm before it starts.

Here is the exact input/output:
```
$ alm --httpskey ../sslkeys/key.pem --httpscert ../sslkeys/server.cert
_tls_common.js:68
      c.context.setCert(options.cert);
                ^

Error: error:0906D06C:PEM routines:PEM_read_bio:no start line
    at Error (native)
    at Object.createSecureContext (_tls_common.js:68:17)
    at Server (_tls_wrap.js:749:25)
    at new Server (https.js:26:14)
    at Object.exports.createServer (https.js:47:10)
    at Object.<anonymous> (/usr/local/lib/node_modules/alm/src/server.js:67:13)
    at Module._compile (module.js:556:32)
    at Object.Module._extensions..js (module.js:565:10)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
    at Module.require (module.js:483:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/usr/local/lib/node_modules/alm/bin/alm:2:1)
    at Module._compile (module.js:556:32)
    at Object.Module._extensions..js (module.js:565:10)
```

It appears as though the files are not actually being read, the file paths are only turned into absolute paths. This causes the crash as `https.createServer` is expecting the actual key and cert values, as shown in https://nodejs.org/api/https.html#https_https_createserver_options_requestlistener

https://github.com/alm-tools/alm/blob/master/src/server.ts#L75
https://github.com/alm-tools/alm/blob/master/src/server/commandLine.ts#L118-L123

**Comment by basarat** [comment]
> the file paths are only turned into absolute paths

Can you log out here : `clOptions` https://github.com/alm-tools/alm/blob/master/src/server.ts#L74 so verify :rose:

**Comment by chase-smith** [comment]
Here is the output.
```
Log for issue #339: { port: 4444,
  dir: '/home/chase/alm_test_dir',
  open: false,
  safe: false,
  project: undefined,
  init: false,
  build: false,
  filePaths: [],
  host: '0.0.0.0',
  httpskey: '/home/chase/sslkeys/key.pem',
  httpscert: '/home/chase/sslkeys/server.cert',
  auth: undefined }
```

**Comment by basarat** [comment]
Sorry my bad. I completely missed reading your comment `It appears as though the files are not actually being read` (my brain parsed it as files are not being made absolute paths). Will fix :rose:

**Comment by basarat** [comment]
Released with 2.6.1 :rose:
